ReactNativeTraining/App.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { FlatList, View } from 'react-native';
+import { List, ListItem } from 'react-native-elements';
+
+import styles from './App.styles';
+import { contacts } from './src/utils/constants';
+
+const App = () => (
+  <List containerStyle={styles.container}>
+    <FlatList
+      ItemSeparatorComponent={() => <View style={styles.greyLine} />}
+      data={contacts}
+      keyExtractor={item => item.id}
+      renderItem={({ item }) => (
```

Comment:
it's bad practice use arrows functions inside `render` function. This cause unwanted re-renders. And why?? Check this. https://medium.freecodecamp.org/why-arrow-functions-and-bind-in-reacts-render-are-problematic-f1c08b060e36
So, convert this components as class component, declare this function inside this class and use it here